Resources/Prototypes/Roles/Jobs/Engineering/chief_engineer.yml

```diff
@@ -40,12 +40,10 @@
 - type: startingGear
   id: ChiefEngineerGear
   equipment:
-    head: ClothingHeadHatHardhatWhite
```

Comment:
I feel a little odd about this one. Just back up your point (which is of course that it's a standard hard hat and therefore not actually directly related to the CE, and that their white colour scheme is still clearly identifiable without it) and I'll probably be fine this.